examples/data-logging-configs/image_classification.yaml

```diff
@@ -4,6 +4,7 @@ data_logging:
   pipeline_outputs.labels:
   - func: predicted_classes
     frequency: 1
+  pipeline_outputs.scores:
```

Comment:
This happens because of the changes in `src/deepsparse/loggers/metric_functions/built_ins.py` below